engines/osxnative/sandboxbuilder.go

```diff
@@ -0,0 +1,33 @@
+package osxnative
+
+import (
+	"github.com/taskcluster/taskcluster-worker/engines"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+type sandboxbuilder struct {
+	engines.SandboxBuilderBase
+	env         map[string]string
+	taskPayload *payload
+	context     *runtime.TaskContext
+}
+
+func (s sandboxbuilder) SetEnvironmentVariable(name string, value string) error {
+	_, ok := s.env[name]
+	if ok {
+		return engines.ErrNamingConflict
+	}
+
+	s.context.Log("Setting \"", name, "\" environment variable to \"", value, "\" ")
+	s.env[name] = value
+	return nil
+}
+
+func (s sandboxbuilder) StartSandbox() (engines.Sandbox, error) {
```

Comment:
It seems that based on this method name of "Start...." that the sandbox (process) should actually be started at this point.

Calling StartSandbox to return a new (unstarted) sandbox seems to go against how this method is named.